piolib/examples/dpi_csync.c

```diff
@@ -0,0 +1,358 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
```

Comment:
Is the copyright date correct or copy pasta?

Reply:
Whoops. It was copied from "dpi_interlace.c". It  looks like several of the examples have it.
Should be 2024 of course - I'll correct it.